templates/manifest.json

```diff
@@ -95,6 +95,40 @@
                 "src/__main__.py"
             ]
         },
+        {
+            "id": "python-empty",
+            "name": "python-empty",
+            "label": "Empty Python project",
+            "category": "python",     
+            "description": "Empty starter Actor. A blank slate with minimal starting code. Useful for quick scripting without having to delete bunch of stuff from other templates.",
+            "archiveUrl": "https://github.com/apify/actor-templates/blob/master/dist/templates/python-empty.zip?raw=true",
+            "defaultRunOptions": {
+                "build": "latest",
+                "memoryMbytes": 1024,
+                "timeoutSecs": 3600
+            },
+            "showcaseFiles": [
+                "src/main.py",
+                "src/__main__.py"
+            ]
+        },
+        {
+            "id": "python-start-scrape-single-page",
+            "name": "python-start-scrape-single-page",
```

Comment:
Pls make sure the IDs are aligned with the other languages. AFAIK they're gonna be `language-start` everywhere, but @jbartadev knows more I guess.

Reply:
Yop, don't worry. We'll unify it before we put it out.